onnx_xla/backend.cc

```diff
@@ -301,21 +293,11 @@ namespace onnx_xla {
     if (handleInputsStatus != ONNXIFI_STATUS_SUCCESS)  {
       return handleInputsStatus;
     }
-
+    auto& registry = OperatorRegistry::registry();
     for (auto it = ir_->begin(); it != ir_->end(); ++it) {
```

Comment:
const auto

Reply:
hm actually doesn't compile with const as no ++ implemented for const_graph_node_list_iterator here https://github.com/onnx/onnx/blob/master/onnx/common/graph_node_list.h